README.zh-CN.md

```diff
@@ -0,0 +1,407 @@
+    - [7.3 清理Demo](#73-清理demo)
+  - [8. 路线图](#8-路线图)
+  - [9. 贡献](#9-贡献)
+  - [10. 许可证](#10-许可证)
+  - [11. 用戶手冊](#11-用戶手冊)
+
+EaseMesh是一个与Spring Cloud生态系统兼容的服务网格。它基于[Easegress](https://github.com/megaease/easegress)开发了Mesh的控制组件和流量边车，并且利用了[EaseAgent](https://github.com/megaease/easeagent)的服务监测，调用链追踪等特性。
+
+<a href="https://megaease.com/easemesh">
+    <img src="./imgs/easemesh.svg"
+        alt="EaseMesh logo" title="EaseMesh" height="175" width="175" align="right"/>
+</a>
+
+## 1. 宗旨
+
+为什么我们要重新发明一个轮子？
+
+- **服务网与Spring Cloud生态系统兼容：** Spring Cloud生态系统中的微服务有自己的服务注册/发现组件。这与使用DNS进行服务发现的Kubernetes生态系统完全不同。主要的服务网格解决方案（如Istio）使用Kubernetes领域技术。但这样的方案会带来许多痛苦，且与Java Spring Cloud生态系统有冲突。EaseMesh旨在使Service Mesh与Java Spring Cloud完全兼容。
+
+- **更细致的观察性：** 目前基于Kubernetes的服务网格只能看到入口/出口流量，它不知道服务/应用中发生了什么。因此，结合Java Agent技术，我们可以拥有观察服务/应用内部和外部的全部能力。
+
+- **复杂的流量分割能力：** EaseMesh具有复杂的流量分割能力，它可以将一个请求链的流量分割到多个服务。这种能力可以应用于金丝雀部署、在线生产测试等场景。
+
+> 简而言之，**EaseMesh利用Kubernetes sidecar和Java Agent技术，使Java应用具有服务治理和细致观察能力，而无需改变任何一行源代码**。
+
+## 2. 原则
+
+- **Spring Cloud兼容性：** Spring Cloud领域服务管理和弹性设计。
+- **无需修改代码：** 使用sidecar和Java Agent来完成服务治理和综合观测。
+- **服务监控：** 服务运行指标/跟踪/日志监控。
+
+## 3. 架构
+
+![The architecture diagram](./imgs/architecture.png)
+
+## 4. 特点
+
+- **非侵入式设计** 对Java Spring Cloud应用程序的迁移不做任何代码修改，只需要进行小规模的配置更新。
+- **Java注册/发现** 与流行的Java Spring Cloud生态系统的服务注册/发现兼容。
+  - **多个租户（命名空间）** 支持多个租户的服务注册，隔离不同租户的服务。
+  - **共享（全局）租户** 支持共享租户，所有服务对全局租户中注册的服务都有可见性。
+  - **兼容性**
+    - 与Eureka注册表兼容。
+    - 与Consul注册表兼容。
+    - 与Nacos注册表兼容。
+  - **可扩展性** 支持用元数据注册服务。
+- **资源管理** 依靠Kubernetes平台来管理CPU/内存资源。
+- **流量编排**
+	- **丰富的路由规则** 精确的路径、路径前缀、路径的正则表达式、方法、头部匹配。
+  - **流量分割** 对东西向和南北向流量进行着色和调度到配置的服务。
+  - **负载平衡** 支持Round Robin, Weight Round Robin, Random, Hash by Client IP Address, Hash by HTTP Headers。
+- **弹性设计** 包括超时/断路/重选/限制器，完全遵循复杂的弹性设计。
+	- **弹性和容错性**
+		- **断路器** 暂时阻止可能的故障。
+		- **速率限制器** 限制传入请求的速率。
+		- **重选器** 重复失败的执行。
+		- **时间限制器** 限制执行的时间。
+  - **混沌工程**
+    - **故障注入** 正在进行的工作。
+    - **延迟注入** 工作正在进行中。
+- **可观察性**
+  - **日志**
+    - **访问日志** 为每个服务的所有请求生成HTTP访问日志。
+    - **应用日志** 自动将跟踪上下文注入到日志数据中。
+  - **追踪**
+    - **JDBC** 对JDBC的调用进行跟踪。
+    - **HTTP请求** 对HTTP RPC的追踪。
+    - **Kafka** 对Kafka传递的消息进行追踪。
+    - **Redis** 追踪Redis缓存的访问。
+    - **RabbitMQ** 跟踪由RabbitMQ传递的消息。
+    - **采样**
+      - 支持概率采样。
+      - 支持 QPS 采样。
+  - **指标**
+    - **HTTP请求** 报告每个URL的吞吐量延迟。
+    - **JDBC** 报告每个SQL的吞吐量和延时。
+    - **Kafka** 报告每个消费者、生产者和主题的吞吐量和延时。
+    - **Redis** 报告每个方法的吞吐量和延时。
+    - **RabbitMQ** 报告每个主题的吞吐量和延时。
+- **安全**
+  - **mTLS** *Working in progress.*
+  - **mTLS强制执行** *Working in progress.*
+  - **外部CA证书** *Working in progress.*
+  - **服务到服务的授权规则** *Working in progress.*
+
+> 吞吐量由m1, m5, m15表示。
+> 延迟由P99、P98、P95、P90、P80、P75、P50等表示。
+
+## 5. 依赖项目
+
+1. [EaseAgent](https://github.com/megaease/easeagent)
+2. [Easegress](https://github.com/megaease/easegress)
+
+## 6. 快速启动
+
+### 6.1 环境要求
+
+- Linux内核版本4.15以上
+- Kubernetes版本1.18以上
+- MySQL版本14.14+
```

Comment:
Invalid mysql version !(should fix it in English version)

Reply:
Updated with 5.7+